src/components/LandingPage.tsx

```diff
@@ -0,0 +1,443 @@
+import {
+  AnnotationIcon,
+  ChatAlt2Icon,
+  InboxIcon,
+  QuestionMarkCircleIcon,
+
+} from '@heroicons/react/outline'
+
+import {
+  GiRetroController,
+  GiMouse,
+  GiBroadsword,
+  GiBrickWall,
+  GiStarsStack
+
+} from "react-icons/gi";
```

Comment:
You don't need the `@heroicons` library. The react-icons contain the all the icons that it has.  
![image](https://user-images.githubusercontent.com/57305093/151041740-4e5c3061-7235-437e-993e-c0e32121ab86.png)